README.md

```diff
@@ -1118,6 +1118,8 @@ They can be overridden for testing purposes in [the same way as other settings](
   ```
 
   **Note**: If `settings.aws.profile` is not set, the setting will fallback to the "default" profile.
+  It is recommended not to use `[profile default]` in the `aws.config` contents.
```

Comment:
Maybe instead of "use", the verb "set" would be more accurate? Since this issue happens if you have [profile default] set in the blob even if you don't use it?

Also I find it weird the sentence right above says the profile setting defaults to "default" and in the next sentence we say not to set it to "default"

Reply:
Reworded slightly to hopefully make it more clear and easier to parse.